src/scroll-timeline-css.js

```diff
@@ -72,16 +73,22 @@ function isDescendant(child, parent) {
 }
 
 function createScrollTimeline(name) {
-  const options = parser.scrollTimelineOptions.get(name);
+  let options = parser.scrollTimelineOptions.get(name) ||
+    parser.getViewTimelineOptions(name);
   if (!options) return null;
 
-  const sourceElement = getSourceElement(options.source);
-
-  const scrollTimeline = new ScrollTimeline({
-    ...(sourceElement ? { source: getSourceElement(options.source) } : {}),
-    ...(options.orientation != "auto" ? { orientation: options.orientation } : {}),
-  });
-  return scrollTimeline;
+  if(options.source) {
+    const sourceElement = getSourceElement(options.source);
+    return new ScrollTimeline({
+      ...(sourceElement ? { source: getSourceElement(options.source) } : {}),
+      ...(options.orientation != "auto" ? { orientation: options.orientation } : {}),
+    });
+  } else {
+    return new ViewTimeline({
+      subject: document.querySelector(options.selector),
```

Comment:
Please add TODO here.  This will catch the first matching selector, which may not be the correct one.  We want the last upstream match.